### ISSUE TITLE
Add OpenSMC Twitter feed to homepage

### ISSUE THREAD

**Issue by ShellSV** [opensmc/opensmc.github.io#14]

**Comment by verythorough** [comment]
Twitter widget builder here: https://twitter.com/settings/widgets/new/user?user_id=4275472874
The Twitter widget builder requires a Twitter account, so for those who might not have one, I've run the builder and generated the embed code:

```
<a class="twitter-timeline" data-dnt="true" href="https://twitter.com/OpenSMC" data-widget-id="715218215090069505">Tweets by @OpenSMC</a>
<script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+"://platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");</script>

```

Copy that code and paste it into `index.html`, at the end of the file, before the comment about adding a twitter widget.  (You can delete the comment, too.)

**Comment by verythorough** [comment]
Closed by #18
